edenai_apis/apis/microsoft/microsoft_ocr_api.py

```
@@ -355,6 +358,119 @@ def ocr__identity_parser(
             standardized_response=standardized_response,
         )
 
+    async def ocr__aidentity_parser(
+        self, file: str, file_url: str = "", model: str = None, **kwargs
```

Ruff (0.14.4)

Unused method argument: `file_url`

(ARG002)